package.json

```
@@ -4,67 +4,14 @@
   "main": "Gruntfile.js",
   "author": "Automattic",
   "scripts": {
-    "semantic-release": "semantic-release",
+    "start": "npm ci",
+    "cm": "newspack-scripts commit",
+    "i18n": "grunt i18n",
+    "readme": "grunt readme",
+    "semantic-release": "newspack-scripts release --files=newspack-new-plugin-boilerplate.php",
```

Small but important :) :

```suggestion
    "semantic-release": "newspack-scripts release --files=republication-tracker-tool.php.php",
```